src/app/pages/datasets/components/delete-dataset-dialog/delete-dataset-dialog.component.html

```diff
@@ -72,7 +77,12 @@ <h1 matDialogTitle>
       ixTest="delete-dataset"
       [disabled]="form.invalid"
     >
-      {{ 'Delete Dataset' | translate }}
```

Comment:
In my opinion, there's no harm in leaving the button text as "Delete".

Reply:
> In my opinion, there's no harm in leaving the button text as "Delete".

good idea